meta/runtime.yml

```diff
@@ -1,2 +1,9 @@
 ---
 requires_ansible: '>=2.9'
+
+action_groups:
+  k8s:
```

Comment:
Shouldn't this be `okd`? Also, better to submit the branch to your fork and then create the PR from there ;)

Reply:
I thought we'd want these grouped with the k8s module so they don't have to replicate the defaults while doing openshift automation with modules from both the k8s and openshift collections.

Reply:
@fabianvf - what's your opinion on this? Do you think it'd be typical that the user would mix k8s and okd module usage together? If so I'm fine with this. I only worry that this could trigger some weird bug in Ansible when it tries to merge the `okd` and `k8s` runtime meta info and `okd` blows out the `k8s` info or something like that (I'm not sure how the action groups code actually would merge the data in two different collections—or if it does at all).

Reply:
I like them using the same `k8s` action group, only caveat being the open questions about how that's merged. If it doesn't break ansible then I think it makes sense for them to share a group

Reply:
Until 2.11 this is all moot anyway, since there's nothing that will load or consult this. 2.10 only shipped with enough of the `action_groups` feature implemented to support a hardcoded collection externalization of the existing short-named groups previously exposed by `module_defaults` in 2.9 (see https://github.com/ansible/ansible/blob/v2.10.1/lib/ansible/executor/module_common.py#L1336-L1348 for the list of names and collections). In this case, the old `k8s` group short name will consist only of modules found in the `k8s` action group definitions in `community.kubernetes` and `community.general`.

The final design for 2.11 is not set in stone, but the intent was that there'd be no "short name" magic for new collection-defined `action_groups`. So the fact that N collections define a group called `k8s` would be irrelevant, because the usage of any new ones would be unambiguous by nature of being fully qualified, eg:
```
module_defaults:
   group/community.okd.k8s:
   (...)
```

So that said, it probably makes more sense that this group be called `okd`, but you can technically call it whatever you want. We've talked about adding redirect and best-effort "include some other action group in this one if you can" capabilities in 2.11, but that's not been settled yet. If those were implemented, and you could convince the owners of either `community.kubernetes` or `community.general` to include your action_group, they'd be merged into the old `k8s` short-name group, but that's a lot of "if"s.

Reply:
So as I understand it, the `action_groups` section of any runtime.yml file has no effect on collections running 2.9 or even the community release of 2.10. Hopefully 2.11 will support such a function, but that is not guaranteed. With 2.11 far off, I may ditch this PR for now.